rest-api-spec/src/main/resources/rest-api-spec/test/update/75_ttl.yaml

```diff
@@ -78,6 +79,28 @@
  - lte:   { _ttl: 20000}
  - gt:    { _ttl: 10000}
 
+# noop
```

Comment:
maybe leave a comment that we are checking that we have this "bug" since it's documented but if it ever gets fixed then we could happily remove this test?

Also maybe it would be better as a Java integration test. I tend to use REST tests for testing the REST API spec but in that case it's more that we want to know if the impl details change?